src/components/Staking/StakingConsiderations/index.tsx

```diff
@@ -75,9 +75,18 @@ const StakingConsiderations = ({ page }: StakingConsiderationsProps) => {
     pageData,
     activeIndex,
   } = useStakingConsiderations({ page })
+  
+  const activeStyles = {
+    bg: "var(--eth-colors-offBackground)",
+    color: "var(--eth-colors-chakra-body-text)",
+    fontWeight: "bold",
```

Comment:
@nloureiro We may want to avoid changing the font weight at all to avoid shifts like this:

https://github.com/ethereum/ethereum-org-website/assets/54227730/b03cc97a-01a6-4ff2-a1b7-220dd15223ae

```suggestion
```

Reply:
![CleanShot 2024-03-24 at 09 42 11](https://github.com/ethereum/ethereum-org-website/assets/57232813/b617c71e-63c6-4a10-ba9a-eb1b6980b3b6)


> @nloureiro We may want to avoid changing the font weight at all to avoid shifts like this:
> 
>  Screen.Recording.2024-03-23.at.15.11.27.mov

Luckily, the new font `fontWeight: "bold"` still works.

Reply:
I've tried many times, and fortunately, `fontWeight: "bold"` works in the latest font. However, such offsets may occur in many situations, such as:

```
      <Flex
        alignItems="center"
        flexDir="column"
        bg="offBackground"
        flex={2}
        minH="410px"
        p={6}
      >
```

Or even in the `IndicatorGroup` section:

```
      <Text
        fontSize="xs"
        textAlign="center"
        width={{ base: "fit-content", sm: "max-content" }}
      >
```

Their main issue is caused by the small resolution. Perhaps the simplest solution is to add a `minW` to the `<ListItem>` (achieving a UI that works flawlessly in every language suddenly feels quite challenging).

If adapting to different resolutions for each language, we might need to use more content-focused squares, such as:

![Frame 1](https://github.com/ethereum/ethereum-org-website/assets/57232813/34ee60b1-82fa-44eb-af45-528262368056)

![Group 7](https://github.com/ethereum/ethereum-org-website/assets/57232813/2124f5e5-1f53-4a2d-aa00-74fa81f958f7)

*I'm not sure if the team will accept this slider design, as I rarely see sliders used on ethereum.org.

The advantage of this design is that it emphasizes content and combines text with images, making it more easily acceptable.

However, the downside is that it requires a lot of illustration designs related to Open Source, Audited, Bug Bounty, Battle-tested, etc., and ensuring consistency in some UI designs.

(For example, replacing emojis with illustrated UI elements.)

It would be great if the team had a concept similar to https://octodex.github.com/. This would aggregate Ethereum's illustration designs and establish a unified visual system, making it easier for other contributors to participate in illustration creation in the future.

Reply:
@iepn I love the design explorations  but I have some considerations and questions.

May I ask you to open an new issue with this ideas so we can discuss them there. 

The scope of this PR is to fix one specific bug and this ideas are more to make a full redesign, let's keep the conversation on a different issue.

Thank you